vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/boutiquesParsing/BoutiquesParser.java

```diff
@@ -114,8 +114,9 @@ public BoutiquesApplication parseApplication(String descriptor) throws InvalidBo
         // Custom property
         JSONObject customObject = getObjectValue(parsedDescriptor, "custom", true);
         if (customObject != null) {
-            String vipImagePath = getStringValue(customObject, "vip:imagepath", true);
-            application.setVipContainer(vipImagePath);
+            application.setVipContainer(getStringValue(customObject, "vip:imagepath", true));
+            application.setVipDotInputIds(getArrayValueAsStringSet(customObject, "vip:dot", true));
+            application.setVipDotResultDirs(getStringValue(customObject, "vip:dot-result-directory", true));
```

Comment:
Rename to `vip:dot-with-results-directory`.
Adapt to `vip:dot-result-directory` being a boolean.

Reply:
change to vip:dot-with-results-directory and adapt to the method getBooleanValue